source/application/BioXAS/BioXAS.h

```diff
@@ -8,12 +8,16 @@ namespace BioXAS
 {
 	/// Handles the XRF detector choice.  Available choices are None, KETEK, and 13-element Ge.
 	enum FluorescenceDetector {
-
 		NoneXRFDetector = 0,
 		FourElementVortexDetector = 1,
 		Ge13ElementDetector = 2
 	};
 
+	/// Enum to describe each of the three BioXAS beamlines.
+	enum Beamline {
```

Comment:
Purely a naming thing, but is Beamline the best description for this enum?  I don't have a better one off the top of my head, but I thought I'd bring it up anyway.